diff -ru Metview-5.15.1-Source-orig/mir/share/mir/masks/CMakeLists.txt Metview-5.15.1-Source/mir/share/mir/masks/CMakeLists.txt
--- mir/share/mir/masks/CMakeLists.txt	2022-03-06 17:31:46.000000000 +0100
+++ mir/share/mir/masks/CMakeLists.txt	2023-09-25 10:59:54.261942630 +0200
@@ -19,7 +19,7 @@
       # get mask into the build dir
       ecbuild_download_resource(
         "${CMAKE_BINARY_DIR}/${_dir}/${_f}"
-        "http://download.ecmwf.org/test-data/mir/${_dir}/${_f}")
+        "https://get.ecmwf.int/repository/mir/${_dir}/${_f}")
 
       # install mask
       install(
